src/components/MoneyReportHeader.tsx

```diff
@@ -247,6 +248,11 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
         [chatReport?.isOwnPolicyExpenseChat, policy?.harvesting?.enabled],
     );
 
+    const shouldDuplicateButtonBeSuccess = useMemo(
+        () => isDuplicate && !shouldShowSettlementButton && !shouldShowExportIntegrationButton && !shouldShowSubmitButton && !hasAllPendingRTERViolations,
```

Comment:
Things to be considered `shouldDuplicateButtonBeSuccess`:
- Looks like the Submit button is not success in employer's end
<img width="775" alt="Screenshot 2024-10-10 at 17 34 32" src="https://github.com/user-attachments/assets/fb3a4e82-f227-4eaf-bec9-d88ef3d25f72">
- I think ExportIntegrationButton will always have default color.

Reply:
I believe so. In that case, should we have Review duplicates button `success`? cc: @dannymcclain

Reply:
🤔 Hmm, maybe? In this case I'm not sure. I don't think it's wrong to have two regular buttons here. But maybe @JmillsExpensify will have a better informed opinion on if the review duplicates button should be `success` colored in this situation.

Reply:
It's not wrong to have two regular buttons. This is the case of delayed submission being enabled, with a submission frequency set.